lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -110,19 +125,29 @@ class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
   }
```

Comment:
I think we want to slide in an extra comma in `));` so it's `),);` and then the Dart formatter will sort it out a bit.